src/golang/cmd/server/routes/routes.go

```diff
@@ -73,6 +74,5 @@ const (
 	GetWorkflowDagResultRoute    = "/api/workflow/{workflowId}/result/{workflowDagResultId}"
 	GetWorkflowHistoryRoute      = "/api/workflow/{workflowId}/history"
 
-	GetServerVersionRoute     = "/api/version"
```

Comment:
Is any SDK code still using this route? To be safe, we could keep these routes and map them to the same handler (similar to workflow trigger / delete / edit routes)

Reply:
Yes, the version route does have a usage. I will keep this one in place.